libs/Database.js

```diff
@@ -1015,6 +1024,50 @@ class Database {
     await this.updateTableHash(table.split('_')[0], table.split('_')[1]);
     await tableInDb.deleteOne({ _id: record._id }, { session: this.session }); // eslint-disable-line no-underscore-dangle
   }
+
+  /**
+   * Used by light nodes to cleanup (unneeded) blocks already verified
+   * by witnesses <= @lastVerifiedBlockNumber
+   * @param lastVerifiedBlockNumber
+   * @returns {Promise<void>}
+   */
+  async cleanupBlocks(lastVerifiedBlockNumber) {
+    if (!this.lightNode) {
+      return;
+    }
+    // await this.chain.deleteMany({ _id: { $lte: lastVerifiedBlockNumber } }, { session: this.session });
```

Comment:
Can you remove this comment?  No need to leave commented code lying around, better to clean it up.